src/allmydata/test/test_openmetrics.py

```diff
@@ -0,0 +1,118 @@
+import mock
+from prometheus_client.openmetrics import parser
+from twisted.trial import unittest
```

Comment:
Can you replace `twisted.trial.unittest.TestCase` in this module with `allmydata.test.common.SyncTestCase`?

Reply:
Not without getting rid of the Mock object it seems, I get a

    TypeError: Object of type Mock is not JSON serializable

Reply:
(@exarkun was so kind to help me with this in a 1:1 meeting, I'll mark this as resolved)